### ISSUE TITLE
Generating LOD lightmap of terrain fails with the nullref

### ISSUE THREAD

**Issue by VergilUa** [guycalledfrank/bakery-issues#11]
`NullReferenceException: Object reference not set to an instance of an object
ftRenderLightmap+<RenderLightmapFunc>d__252.MoveNext () (at Assets/Editor/x64/Bakery/scripts/ftRenderLightmap.cs:5614)
ftRenderLightmap.RenderLightmapUpdate () (at Assets/Editor/x64/Bakery/scripts/ftRenderLightmap.cs:3743)`

This is on Unity 2019.3.7f1 + HDRP 7.1.8. Bakery v.1.71, bake had following settings:
![image](https://user-images.githubusercontent.com/24496266/79871647-541e9680-83ed-11ea-8499-7e1af115f833.png)

**Comment by guycalledfrank** [comment]
Hi, are you sure it's the full error message? Can't see anything specific on these lines. Is there also any error message starting with "Exception caught:"?  It should have information on the deepest responsible line number.

**Comment by VergilUa** [comment]
Hi, it's not. Stacktrace leads to a bunch of coroutine runing invokable functions (its the last useful line out there), so I've truncated it a bit. 

Yes, LOD lightmap is mentioned in the progress bar.

**Comment by guycalledfrank** [comment]
Right... it's odd that it doesn't point to the actual problematic line number.
LOD lightmaps are generated when GI VRAM optimization is on: https://geom.io/bakery/wiki/index.php?title=Manual#GI_VRAM_optimization

Which is on when estimated VRAM usage is higher than available. It's not well (enough) tested for the obvious reason of users not willing to provide any large scenes for debugging and this feature mostly only kicking in on large scenes.

Bakery works in a much more predictable fashion when this feature is off, but it will require you to reduce resolution (texels per unit/scale in lightmap) or get more VRAM.

It would help if you show me a full screenshot of all debug messages (not just errors) prior to the final error.
